### DISCUSSION TITLE
Identification of complex models：the following blocks (ids) do not exist on the mesh

### DISCUSSION THREAD

**Discussion by Krystalfang** [idaholab/moose#21492] [Q&A General] [answered]
My model is more complicated than cylinders, etc. Models can only be imported while being processed. I completed the pre-processing in Trelis, defined blocks, sidesets, and divided grids, but during the execution of moose, the same error the following blocks ( ids) do not exist on the mesh: pla (65535).
![image](https://user-images.githubusercontent.com/107802065/177155595-734d8757-15b2-45a1-8bf6-4a6da34393bd.png)
But I do have  defined block: pla 
![3 BYH4NZA % YCX`JS{D${B](https://user-images.githubusercontent.com/107802065/177155805-bb3c3d5d-0322-4c8b-91f2-b3bf508f8bb8.png)
My input file should be runnable because I strip out the complex parts of the model and it works.
It is very strange that I sometimes re-process the pre-processing, and it is possible to perform iterative calculations on moose, but after changing the mesh or other operations on this basis, similar problems will occur.
What I want to ask is whether this problem is caused by the loss of mesh file information caused by complex models, and where should I start to solve this problem.

**Comment by GiudGiud** [comment, marked as answer]
Hello

Which object is returning this error?
How many blocks do you have in the mesh exactly?

Can you share your mesh file? Might be easier to look at it directly. 

Guillaume

**Reply by GiudGiud** [reply]
Solved off-forum (by email).
pla block did not exist after Trelis output the exodus as it was replaced by pla_TETRA and pla_PYRAMID in the exodus file because it contained mixed element
